aruco settings:
  squares_horizontally: 5
  squares_vertically: 7
  square_length: 0.03
  marker_length: 0.5

image_topic: "/usb_cam/image_raw"